src/app/pages/create-wallet/create-wallet.scss

```diff
@@ -0,0 +1,92 @@
+@import 'preact-material-components/Card/mixins';
+@import 'preact-material-components/Button/mixins';
+@import '../../styles/base';
+
+$mdc-layout-grid-breakpoints: (
```

Comment:
i don't think we should change the breakpoints for the grid, at least not with this values :)
defaults are:
```
desktop: 840px,
tablet: 480px,
phone: 0px
```
i think these values are ok.